doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,471 @@
+  * Samples are using :option:`CONFIG_NVS` instead of :option:`CONFIG_FCB` as the default storage backend.
+  * Fixed an issue in :ref:`bt_mesh_light_ctrl_srv_readme` by always setting the transition time to a Fade Standby Manual state time when a Light Off event is received.
+  * Fixed an issue by reporting maximum remaining time for all components for CTL state transition time when GET is processed.
+  * Fixed an issue where a deleted :ref:`bt_mesh_scene_srv_readme` did not delete all its pages from the file system.
+  * Fixed an issue where Sensor Threshold was trimmed and an invalid value was calculated.
+  * Updated :ref:`bt_mesh_scheduler_srv_readme` to no longer extend :ref:`bt_mesh_scene_srv_readme`, but they must be present on the same element.
+  * Moved the configuration settings for acknowledged messages into Kconfig to make them public.
+  * Fixed an issue where an Occupancy On event did not transition to a Fade On state even if Occupancy Mode is disabled.
+  * Added a flag to :ref:`bt_mesh_onoff_srv_readme` to skip Default Transition Time on Generic OnOff Set (Unack) messages.
+  * Fixed an issue by correcting the bindings between the Generic OnOff state and the Light OnOff state.
+  * Fixed an issue by clearing the internal sum in the proportional-integral (PI) feedback regulator when entering the OFF state of the state machine.
+  * Fixed an issue where :ref:`bt_mesh_lightness_srv_readme` could accidentally disable :ref:`bt_mesh_light_ctrl_srv_readme` before it was started.
+  * Fixed an issue by publishing the Light Lightness Status message on startup even if the OnPowerUp state is OFF.
+  * Fixed issues by publishing the Light OnOff Status when disabling and restoring the Light LC state.
+  * Fixed an issue where temperature and range should be within a valid default range for the :ref:`bt_mesh_light_temp_srv_readme`.
+  * Removed sensor type ``BT_MESH_SENSOR_DELTA_DISABLED`` as it is removed from specification.
+  * Replaced ``struct bt_mesh_model_ack_ctx`` with ``struct bt_mesh_msg_ack_ctx`` from :ref:`zephyr:bluetooth_mesh_msg`.
+
+Trusted Firmware-M
+------------------
+
+* Added support for hardware-accelerated cryptography in TF-M using the Nordic Security module (nrf_security).
+  When enabled (default), any calls to psa_crypto APIs will utilize the CryptoCell hardware on nRF9160 and nRF5340.
+* Added support for using hardware unique keys (HUKs) for key derivation (``TFM_CRYPTO_ALG_HUK_DERIVATION``).
+  TF-M automatically generates and stores random hardware unique keys (if not present), using the :ref:`lib_hw_unique_key` library.
+* TFM_MINIMAL: Added a size-optimized configuration of TF-M which provides a minimal set of features:
+
+  * This configuration requires 32 kB of flash and provides random number generation, SHA-256, and the platform memory read service, which is analogous to the feature set of :ref:`secure_partition_manager`.
+  * The configuration is showcased in the :ref:`tfm_hello_world` sample.
+
```

Comment:
```suggestion
* The TF-M protected storage service is now using non-static labels when deriving encryption keys.
  The derivation labels are generated by combining the client id of the requesting service and the UID of the resource.
* It is now possible to run PSA tests (psa-arch-tests) and TF-M regression tests in NCS.
  The tests can be found as samples in zephyr/samples/tfm_integration.

```

Reply:
@Vge0rge @ioannisg  ^

Reply:
That seems great, and maybe we can add a comment regarding the nRF9160 configuration:
The TF-M protected storage on nRF9160 is now configured to use AES CCM to perform authenticated encryption/decryption.

Reply:
added